ignite/services/plugin/template/integration/integration_test.go.plush

```diff
@@ -0,0 +1,71 @@
+package explorer_test
```

Comment:
why the package name is called like that?

Reply:
A mistake from me!